README.md

```diff
@@ -1 +1,5 @@
-# Hello-Microverse-2
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please follow this [template](https://github.com/microverseinc/readme-template) for your readme file to make it more professional

Reply:
- [Optional ] The review is polite and clear as well as neutral. 👍 👍